aea/test_tools/test_cases.py

```diff
@@ -344,6 +363,20 @@ def generate_wealth(cls, ledger_api_id: str = FETCHAI_NAME) -> None:
             "generate-wealth", ledger_api_id, "--sync", cwd=cls._get_cwd()
         )
 
+    @classmethod
+    def get_wealth(cls, ledger_api_id: str = FETCHAI_NAME) -> str:
+        """
+        Get wealth with CLI command.
+        Run from agent's directory.
+
+        :param ledger_api_id: ledger API ID.
+
+        :return: command line output
+        """
+        cls.run_cli_command("get-wealth", ledger_api_id, cwd=cls._get_cwd())
+        assert cls.last_cli_runner_result is not None, "Runner result not set!"
+        return str(cls.last_cli_runner_result.stdout_bytes, "utf-8")
```

Comment:
This does not work..which is odd.

Reply:
@marcofavorito any ideas?

Reply:
could you give more details? Can you help me to reproduce the issue?

Reply:
Easy to reproduce, run erc1155 test script and place debugger after the command is called (currently commented out).